tests/cli/test_compose_qcow2.sh

```diff
@@ -51,31 +51,26 @@ __EOF__
 
         rlRun -t -c "$CLI blueprints push $TMP_DIR/with-ssh.toml"
 
-        UUID=`$CLI compose start with-ssh qcow2`
+        UUID=$($CLI compose start with-ssh qcow2)
         rlAssertEquals "exit code should be zero" $? 0
 
-        UUID=`echo $UUID | cut -f 2 -d' '`
+        UUID=$(echo "$UUID" | cut -f 2 -d' ')
     rlPhaseEnd
 
     rlPhaseStartTest "compose finished"
-        wait_for_compose $UUID
-        rlRun -t -c "$CLI compose image $UUID"
-        IMAGE="$UUID-disk.qcow2"
-    rlPhaseEnd
-
-    rlPhaseStartTest "Start VM instance"
-        boot_image "-boot c -hda $IMAGE" 60
-    rlPhaseEnd
-
-    rlPhaseStartTest "Verify VM instance"
-        # run generic tests to verify the instance
-        verify_image root localhost "-i $SSH_KEY_DIR/id_rsa -p $SSH_PORT"
```

Comment:
Same comment here for booting the test VM and verifying the VM instance. It is important to preserve existing behavior and be able to execute these scripts by hand, like the rest of them.